### PR TITLE
Slimepeople can now morph the shape and size of their genitalia.

### PR DESCRIPTION
**Pull request by Sishen1542**
## About The Pull Request
-Added a few more functions to the slimeperson form alteration ability, giving easy access to change genitalia sizes/shapes.
-Fixed a wrong proc call in alter form for removing genitalia.

-Thanks to Pooj for helping me figure some things out.

## Why It's Good For The Game
Slimepeople can already alter their appearances, why not be able to alter your naughty bits as well?

## Changelog
:cl:
add: Added the ability to alter your genitalia as a slimeperson more than addition/removal.
fix: fixed genitalia removal proc in alter form.
/:cl:

### CODE REVIEW

**Comment by RogueSteampunker** [comment]
The absolute madlad did it

File: modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 1]
```diff
@@ -194,5 +194,51 @@
 			if(new_taur != "None")
 				H.dna.features["mam_tail"] = "None"
 		H.update_body()
+
+	else if (select_alteration == "Penis")
```
Comment (Poojawa) [inline modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm:198]: Reminder that you need balls to orgasm with this

File: modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 2]
```diff
@@ -194,5 +194,51 @@
...
+		var/new_shape
+		new_shape = input(owner, "Choose your character's dong", "Genital Alteration") as null|anything in GLOB.cock_shapes_list
+		if(new_shape)
+			H.dna.features["cock_shape"] = new_shape
+		H.update_genitals()
+		H.give_penis()
+		H.apply_overlay()
+
+
+	else if (select_alteration == "Vagina")
```
Comment (Poojawa) [inline modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm:211]: reminder that you need a womb to orgasm with this.

File: modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 3]
```diff
@@ -194,5 +194,51 @@
...
+			qdel(X)
+		var/new_length
+		new_length = input(owner, "Penis length in inches:\n([COCK_SIZE_MIN]-[COCK_SIZE_MAX])", "Character Preference") as num|null
+		if(new_length)
+			H.dna.features["cock_length"] = max(min( round(text2num(new_length)), COCK_SIZE_MAX),COCK_SIZE_MIN)
+		H.update_genitals()
+		H.apply_overlay()
+		H.give_penis()
+
+	else if (select_alteration == "Breast Size")
```
Comment (Poojawa) [inline modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm:235]: Breasts have different shapes as well.

**Review by Poojawa — CHANGES_REQUESTED** [review]

**Comment by Sishen1542** [comment]
Added balls/womb procs to penis/vagina respectively, also added breast shape option per Poojawa's request.

**Comment by Tupinambis** [comment]
https://www.youtube.com/watch?v=dGDH3meSPyk

**Review by Poojawa — APPROVED** [review]

File: modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 4]
```diff
@@ -194,5 +194,65 @@
...
+	else if (select_alteration == "Penis")
+		for(var/obj/item/organ/genital/penis/X in H.internal_organs)
+			qdel(X)
+		var/new_shape
+		new_shape = input(owner, "Choose your character's dong", "Genital Alteration") as null|anything in GLOB.cock_shapes_list
+		if(new_shape)
+			H.dna.features["cock_shape"] = new_shape
+		H.update_genitals()
+		H.give_balls()
+		H.give_penis()
```
Comment (deathride58) [inline modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm:207]: This gives genitals even if you select "cancel", which is interpreted as null. This means that that the ability cannot be used to remove your genitals. Add a way to remove genitals via these prompts. Ditto for all of the other genital prompts here

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm (hunk 4 above)
Comment (Sishen1542) [reply modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm:207]: Addition/removal is handled by the genitals option above, this was just to allow people to orgasm with said new bits.

**Review by deathride58 — APPROVED** [review]

**Pull request merged**